ambry-store/src/main/java/com.github.ambry.store/BlobStoreStats.java

```diff
@@ -355,6 +357,10 @@ public void close() {
       }
       metrics.statsOnDemandScanTimePerIndexSegmentMs.update(time.milliseconds() - indexSegmentStartProcessTimeMs,
           TimeUnit.MILLISECONDS);
+      indexSegmentCount++;
+      if (indexSegmentCount == 1 || indexSegmentCount % 10 == 0) {
+        logger.info("Index segment processing(on-demand scanning) for container stats complete for {} " + indexSegment);
```

Comment:
there is a "+" here that needs to be removed. Please take a look and verify all statements.

Also, a space is required b/w "processing" and the opening parentheses